doc/hardware/peripherals/mipi.rst

```diff
@@ -0,0 +1,13 @@
+.. _mipi_api:
+
+MIPI
```

Comment:
`MIPI-DSI` (MIPI has other standards, e.g. MIPI-CSI2). same for file name (`mipi_dsi.rst`)

Reply:
Renamed to `mipi_dsi`